utils/base_api.py

```diff
@@ -0,0 +1,118 @@
+import logging
+from typing import Callable
+
+import jsonschema as jsonschema
+import requests
+from deepdiff import DeepDiff
+
+
+class BaseApi:
+    def __init__(self):
+        self.__api_name = 'Petstore'
+        self.__base_url = 'https://petstore.swagger.io/v2'
+
+    @staticmethod
```

Comment:
I do not think this decorator is a part of the BaseApi class. It should be outside it.